pkg/v1/tkg/test/framework/cluster_proxy.go

```diff
@@ -88,7 +88,7 @@ func (p *ClusterProxy) GetScheme() *runtime.Scheme {
 // GetClusterNodes gets the cluster Nodes
 func (p *ClusterProxy) GetClusterNodes() []corev1.Node {
 	clientSet := p.GetClientSet()
-	nodeList, err := clientSet.CoreV1().Nodes().List(metav1.ListOptions{})
+	nodeList, err := clientSet.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
```

Comment:
Use `context.Background()` instead, unless a context can be derived from a parent context.

Reply:
Done.